osquery/tables/system/darwin/system_extensions.mm

```diff
@@ -48,6 +48,62 @@ void getExtensionRow(const pt::ptree& extension, Row& r) {
   const auto category = extension.get_child("categories");
   r["category"] = getExtensionCategory(category);
   r["bundle_path"] = extension.get("container.bundlePath", "");
+  r["is_managed"] = INTEGER(0);
+}
+
+// com.apple.system-extension-policy payload defines the policy for
+// the system extensions. The function looks for team identifier or
+// extension bundle identifiers in the allowed list.
+// https://developer.apple.com/documentation/devicemanagement/systemextensions?language=objc
+void setPolicyFlag(const pt::ptree& ptree, Row& r) {
```

Comment:
Nitpick, but, can we call this `getPolicyFlag` rather than `setPolicyFlag`? I know that we are generating the value for the virtual table, but we are not setting or changing anything on the system.